### PR TITLE
Major refactoring sequana_coverage

### PR DESCRIPTION
**Pull request by cokelaer**
Other fixes:

- uses click for sequana_coverage, sequana rnadiff and sequana_taxonomy
- fix circular update to use latest multiqc
- improve upsetplot to cope with large set of comparisons
- update special cases for gff/gbk
- add pipeline version in summart, wrappers versions in HTML

### CODE REVIEW

**Review by rania-o — APPROVED** [review]

File: sequana/modules_report/rnadiff.py [hunk 1]
```diff
@@ -190,11 +190,19 @@ def add_cluster(self):
...
+            with pylab.ioff():
+                pylab.clf()
+                import warnings
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore")
+                    self.rnadiff.plot_dendogram()
+                    # Here, tight layout is not comaptible with dendogram
+                    # Yet, if we call it and plot again, we get the expected layout.
+                    # otherwise, colormap is kind of shifted and not as large as the plot itself.
+                    pylab.tight_layout()
```
Comment (khourhin) [inline sequana/modules_report/rnadiff.py:202]: Old "tight_layout"

File: sequana/scripts/main/lane_merging.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 from snakemake import shell as shellcmd
 
 import click
```
Comment (khourhin) [inline sequana/scripts/main/lane_merging.py:22]: Out of curiosity, do we still need the `import click` ?

File: sequana/scripts/main/taxonomy.py [hunk 3]
```diff
@@ -10,7 +10,9 @@
 #
 ##############################################################################
 import click
+import rich_click as click
 import colorlog
+import rich_click as click
```
Comment (khourhin) [inline sequana/scripts/main/taxonomy.py:16]: 2 rich_click

**Review by khourhin — APPROVED** [review]

File: sequana/modules_report/rnadiff.py (hunk 1 above)
Comment (cokelaer) [reply sequana/modules_report/rnadiff.py:202]: fixed indeed.

**Pull request merged**